map_test.py

```diff
@@ -0,0 +1,112 @@
+import interfaces
```

Comment:
This file should probably be moved into the integrations test folder. Or at least just somewhere else